probs/probs.go

```diff
@@ -107,6 +108,16 @@ func Conflict(detail string, a ...interface{}) *ProblemDetails {
 	}
 }
 
+// AlreadyRevoked returns a ProblemDetails with a AlreadyRevokedProblem and a 409 Conflict
+// status code.
+func AlreadyRevoked(detail string, a ...interface{}) *ProblemDetails {
+	return &ProblemDetails{
+		Type:       AlreadyRevokedProblem,
+		Detail:     fmt.Sprintf(detail, a...),
+		HTTPStatus: http.StatusConflict,
```

Comment:
Based on the spec this looks like it should be `http.StatusBadRequest` (`For example, if the certificate has already been revoked the server returns an error response with status code 400 (Bad Request) and type "urn:ietf:params:acme:error:alreadyRevoked".`).

Reply:
Oops. I actually already changed it, but apparently forgot to push it...

Reply:
Sorry!

Reply:
No worries! Thanks.